src/components/ArtifactQualityLabel/ArtifactQualityLabel.tsx

```diff
@@ -51,7 +43,5 @@ interface IArtifactQualityLabelProps {
 }
 
```

Comment:
Move all specialized mappers like `ArtifactQualityLabel.tsx` , `ArtifactRepositoryTypeLabel.tsx` , etc. into the `components/Label/` folder. Example: `components/Label/ArtifactQualityLabel.tsx`

Reply:
done